docker/README.md

```diff
@@ -54,9 +51,77 @@ Once set up, ensure your environment is set up with `eval %{docker-machine env $
 
 Run the [./docker/deploy.sh](deploy.sh) script as in the previous example.
 
-In order to automatically launch the 1Password SCIM bridge upon startup when using `docker-compose`, you will need to configure systemd to automatically start the Docker daemon and launch op-scim.
 
-1. Install the service file for op-scim. A [sample](compose/op-scim.service) is provided and you'll need to modify it to your needs.
-2. Once installed, reload systemd so that it recognizes the service: `systemctl daemon-reload`
-3. Enable the op-scim service: `systemctl enable op-scim`
-4. Start the op-scim service: `systemctl start op-scim`
+### Manual Instructions
+
+#### Creating the `scim.env` file
```

Comment:
I feel like we should create this file with lots of comments in it and encourage the user to edit it

Reply:
I'll leave that to another PR, since I think that'll take a bit more work in terms of documentation.